src/bin/lockin.rs

```diff
@@ -18,16 +18,34 @@ use stabilizer::hardware::{
 use miniconf::Miniconf;
 use stabilizer::net::{Action, MqttInterface};
 
+// A constant sinusoid to send on the DAC output.
+// Full-scale gives a +/- 10.24V amplitude waveform. Scale it down to give +/- 1V.
+const ONE: i16 = ((1.0 / 10.24) * u16::MAX as f32) as _;
+const SQRT2: i16 = (ONE as f32 * 0.707) as _;
+const DAC_SEQUENCE: [i16; design_parameters::SAMPLE_BUFFER_SIZE] =
+    [ONE, SQRT2, 0, -SQRT2, -ONE, -SQRT2, 0, SQRT2];
+
 #[derive(Copy, Clone, Debug, Deserialize, Miniconf)]
 enum Conf {
-    PowerPhase,
+    Power,
+    Phase,
+    PllFrequency,
     FrequencyDiscriminator,
```

Comment:
@jordens Do you have better names than `PllFrequency` and `FrequencyDescriminator`? I don't know if the names I selected make sense for these measurements (as I'm not quite sure what these values are).

Reply:
`InPhase` (`.re`), `Quadrature` (`.im`), `Magnitude` (`abs_sqr()`), `LogPower` (`log2()`), `Phase` (`arg()`), `PllFrequency` (`pll_frequency`), `Modulation` (`DAC_SEQUENCE`) should be the ones. Each channel should support all of them, independently.

Reply:
Alright, should now be cleaned up. Thanks for the input!